addons/calendar/models/calendar_event.py

```diff
@@ -355,6 +355,43 @@ def _compute_attendee(self):
             attendee = mapped_attendees[meeting.id]
             meeting.attendee_status = attendee.state if attendee else 'needsAction'
 
+    @api.model
+    def get_search_attendee_status(self, operator, value):
+        negative = operator in expression.NEGATIVE_TERM_OPERATORS
+        all_status = set([status[0] for status in Attendee.STATE_SELECTION])
+
+        # In case we have no value or non supported operator
+        if not value or operator not in ['=', '!=', 'in', 'not in']:
+            return all_status if negative else []
+
+        if operator == '=':
+            return {value}
+        elif operator == '!=':
+            return all_status - {value}
+        elif operator == 'in':
+            return value
+        elif operator == 'not in':
+            return all_status - set(value)
```

Comment:
This is not working because when operator is `!=` value is string so we can do {value} to convert it to set using `set(value)` will make  set of its charecters And for `not in` we need to use `set` method because doing `{value}` does not change its type from list to set :slightly_smiling_face: